src/modules/menu/components/mainMenu/MainMenu.js

```diff
@@ -121,6 +121,11 @@ export class MainMenu extends MvuElement {
 			container.style.width = parseInt(event.target.value) + 'em';
 		};
 
+		const getValue = () => {
+			const container = this.shadowRoot.getElementById('mainmenu');
+			return (container && container.style.width !== '') ? parseInt(container.style.width) : 28;
```

Comment:
seems to be a magic number... 
let's move it to a static getter and add a test on this method (see SWIPE_DELTA_PX for an example)
```
static get INITIAL_WIDTH_EM() {
		return 28;
	}
```

Reply:
done